README.md

```diff
@@ -50,12 +50,15 @@ After completing ALL of the above, remove this `Project bootstrap` section from
 ### Setup
 - Inside the `backend` folder, do the following:
   - Create a copy of `{{project_name}}/settings/local.py.example`:  
-  `cp {{project_name}}/settings/local.py.example {{project_name}}/settings/local.py`
+  `cp {{project_name}}/settings/local.py.example {{project_name}}/settings/local.py`
   - Create a copy of `.env.example`:
   `cp .env.example .env`
 
 ### If you are using Docker:
-- Open a new command line window and go to the project's directory.
+- Open the `/backend/.env` file on a text editor and uncomment the line `DATABASE_URL=postgres://{{project_name}}:password@db:5432/{{project_name}}`
```

Comment:
When using Docker, can you only use postgres or sqllite works as well?

Reply:
Actually, yes. It'd be the same as it's now, the user would just have to choose the other env var and it'd create the sqlite file. The postgres container would still be up, though.